docs/v2/api-endpoints/validate_tx_out.md

```diff
@@ -0,0 +1,60 @@
+# Validate TxOut
+
+Allows the public key of a tx out to be checked against the ledger, and if it exists will return the block index
+
+## Request
+
+| Param | Description |  |
+| :--- | :--- | :--- |
+| `public_key` | The public key of the txo. | public key is hex encoded bytes |
+
+## Response
+
+## Example
+
+{% tabs %}
+{% tab title="Request Body" %}
+```text
+{
+    "method": "validate_tx_out",
```

Comment:
I think the name validate txout is problematic since no validation is taking place... how about `"get_tx_out_block_index"`?